docs/rules/valid-typeof.md

```diff
@@ -20,7 +20,7 @@ Examples of **incorrect** code for this rule:
 typeof foo === "strnig"
 typeof foo == "undefimed"
 typeof bar != "nunber"
-typeof bar !== "fucntion"
+typeof bar !== "function"
```

Comment:
@platinumazure @jeis2497052 I think this is wrong. These are deliberate typos, aren't they?

Reply:
Agreed, would you like to create a PR to fix this example?

Reply:
#10459 :)